src/data/matrix/basic.lean

```diff
@@ -878,6 +882,12 @@ by { ext, refl }
 
 end transpose
 
+section has_star
+variables [has_star α]
+instance : has_star (matrix n n α) := {star := conj_transpose}
+lemma star_eq_conj_transpose (M : matrix m m α) : star M = Mᴴ := rfl
+end has_star
```

Comment:
Can you move the `star_apply` lemma on line 904 to this section too?

Reply:
This is now done.